src/lib/common/file/file-upload.directive.ts

```diff
@@ -54,11 +56,21 @@ export class FileUploadDirective extends NgComponent implements AfterViewInit {
 		this.listener.on('change', this.upload);
 		this.listener.on('click', this.onClick);
 		this.listener.on('drop', this.upload);
+		this.listener.on('focus', this.onFocus);
 
 		this.reader.onloadend = e => this.setDataUrl(e);
 	}
 
+	onFocus(e) {
+		if (this.backdrop.hidden) {
+			this.revealBackdrop();
+		}
+	}
+
 	onClick() {
+		this.backdrop = document.querySelector('.q-grid-backdrop');
```

Comment:
what if there won't be a backdrop? e.g. user rewrites cell template and doesn't use backdrop?